qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -109,12 +109,13 @@ def change_virtio_media(cdrom_virtio):
                                         installer_pkg_check_cmd,
                                         copy_files_params=params)
 
-    error_context.context("Run viofs service after upgrade...")
-    win_driver_installer_test.run_viofs_service(test, params, session)
-
```

Comment:
Hi @xiagao Why we need to remove these 2 line?Seems we only need to add `run_viofs_service` after vm reboot.

Reply:
I did't remove, pls look at the following lines,just move it below.

Reply:
We used to run `run_viofs_service` first and the reboot the vm,but now we run `run_viofs_service` after vm reboot.

Reply:
This patch indeed modify 'start virtiofs after reboot guest' issue, before we didn't consider it because most of windows guests don't need to reboot, but for win2012, it will reboot guest, but after reboot, there is no start virtiofs steps.